pkg/daemon/rpm-ostree.go

```diff
@@ -168,14 +174,16 @@ func (r *RpmOstreeClient) PullAndRebase(container string, keep bool) (imgid stri
 		args = append(args, container)
 		pivotutils.RunExt(false, numRetriesNetCommands, "podman", args...)
 	} else {
-		var targetMatched bool
-		targetMatched, err = compareOSImageURL(previousPivot, container)
-		if err != nil {
-			return
-		}
-		if targetMatched {
-			changed = false
-			return
+		if previousPivot != "" {
+			var targetMatched bool
+			targetMatched, err = compareOSImageURL(previousPivot, container)
+			if err != nil {
+				return
```

Comment:
log this err?

Reply:
Not new in this PR; had to look, but this function uses named return values, so we're correctly propagating errors back up to the caller, who will handle it (usually log).

Reply:
awesome, missed that, sorry for the noise